w08h02/test/pgdp/pools/structural/TaskFunctionStructuralTest.java

```diff
@@ -0,0 +1,37 @@
+package pgdp.pools.structural;
+
+import java.lang.reflect.Modifier;
+import java.util.function.Function;
```

Comment:
```suggestion
```

This import is unused, right?

Reply:
Nope, Function is used in line 20 and 26 ^^

Reply:
Ah wrong file, sorry